.github/ISSUE_TEMPLATE/bug-template.md

```diff
@@ -0,0 +1,25 @@
+Thank you for taking the time to let us know about the issue you found. Please provide us with the information below and we will look at it as soon as we are able.
+
+## Description
+
+*Please provide a short description of the bug*
+
+## Action Taken
+
+*In what way were you interacting with the application when you discovered the issue?*
+
+## What I expected to see
+
+*Please provide a short description of what you expected to see*
+
+## What I did see
+
+*Please provide a short description of what you did see. Screenshots are helpful, but please block out any personally identifying information before posting.*
+
+## Other Helpful Information
+
++ URL of the page I was on:
++ Browser and version:
```

Comment:
this could be overkill, but should we include examples here too (e.g. Windows, MacOS) ?

Reply:
I had considered it when I made the template. I don't mind adding it here.